### PR TITLE
Remove docker-compose file and update Readme

### PR DESCRIPTION
**Pull request by mats-claassen**
Also remove confusing use of `inference_command`

### CODE REVIEW

**Comment by aelkhour** [comment]
@mats-claassen Out of curiosity, what are the pros of removing the docker-compose file? I feel like the replacement command is a bit complex, and I liked having a simple command to launch the inference server without having to look it up in the README.

File: README.md [hunk 1]
```diff
@@ -187,23 +188,19 @@ detected for that image).
...
 ### Build and run the mock inference service container
 
 ```bash
-# Start the service
-docker-compose up -d
+# Build the docker image (run once)
+docker build -t arterys_inference_server .
+
+# Start the service.  
+docker run --gpus=all --rm -v $(pwd):/opt -p 8900:8000 -d arterys_inference_server <command>
```
Comment (bsblaw) [inline README.md:203]: Let's remove the --gpu=all here since the mock server doesn't need it.

**Comment by bsblaw** [comment]
> @mats-claassen Out of curiosity, what are the pros of removing the docker-compose file? I feel like the replacement command is a bit complex, and I liked having a simple command to launch the inference server without having to look it up in the README.

The mock server was being modified to take a mandatory input arg now, and instead of passing the arg through to docker-compose, we thought we would get rid of a layer of indirection and just pass the arg directly to docker run. Could go either way really.

**Comment by aelkhour** [comment]
> > @mats-claassen Out of curiosity, what are the pros of removing the docker-compose file? I feel like the replacement command is a bit complex, and I liked having a simple command to launch the inference server without having to look it up in the README.
> 
> The mock server was being modified to take a mandatory input arg now, and instead of passing the arg through to docker-compose, we thought we would get rid of a layer of indirection and just pass the arg directly to docker run. Could go either way really.

Thanks for the explanation @bsblaw :-)
As a developer with little docker wizardry experience, I enjoyed having an easy-to-read README with simple commands that I can remember. This might be just me, but I feel the indirection could be helpful for other newcomers.

**Pull request merged**